packages/java-shell/package.json

```diff
@@ -11,13 +11,12 @@
     "url": "git://github.com/mongodb-js/mongosh.git"
   },
   "scripts": {
-    "browserify-shell-api": "browserify -g uglifyify src/main/js/all.js -o src/main/resources/js/all-standalone.js -dv",
+    "webpack-shell-api": "webpack --mode development",
     "test": "mocha -r \"../../scripts/import-expansions.js\" --timeout 60000 --colors -r ts-node/register \"./src/test/js/run-tests.ts\"",
     "test-ci": "node ../../scripts/run-if-package-requested.js npm run test"
   },
   "devDependencies": {
-    "browserify": "latest",
-    "uglifyify": "latest"
+    "browserify": "latest"
```

Comment:
Should this be webpack instead of browserify now? Not sure if this matters at all, as the base config we seem to require right from the root 🤔

Reply:
This was just a lazy way of getting all the browserify polyfills in place :slightly_smiling_face: Switched to an explicit list in 069806dcf1fb0c9d56d2d4c8c30dcf0ea79dd4ae